src/ui/constants/visualizations/scatter.ts

```diff
@@ -118,6 +118,6 @@ export const SCATTER_VISUALIZATION: GraphShared['visualization'] = {
 export const SCATTER_D3_VISUALIZATION: GraphShared['visualization'] = {
     ...SCATTER_VISUALIZATION,
     id: WizardVisualizationId.ScatterD3,
-    allowShapes: false,
+    allowShapes: true,
```

Comment:
I think it is possible to delete a line as in LINE_D3_VISUALIZATION? So that you can see how the visualizations differs